src/App.js

```diff
@@ -1,29 +1,16 @@
-import logo from './logo.svg';
+import React from 'react';
```

Comment:
According to the requirements:

![image](https://user-images.githubusercontent.com/57421823/171382455-f79a7a38-6868-4b51-94a7-c29b8d7e9de2.png)

Therefore:

- [ ] Please, delete all the boilerplate unused code that was left by the CRA.  This includes text, styles, and images.  Almost all of them are on the "public" directory.

Reply:
Hi @Meltrust, thank you for the code review.